prototype/core/geo-json/src/main/java/org/eclipse/sensinact/gateway/geojson/GeoJsonObject.java

```diff
@@ -61,4 +63,44 @@ protected GeoJsonObject(GeoJsonType type) {
     public Map<String, Object> foreignMembers = new HashMap<>();
 
     public List<Double> bbox;
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(type, bbox);
+    }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
I don't think this should be an override of `equals` as it's not a complete equals method. 

If it were:

    protected boolean checkParentEquals(Object obj) {

Then that would prevent any confusion that this might be usable on its own.